spec/features/user_registers_spec.rb

```diff
@@ -1,12 +1,60 @@
 require 'rails_helper'
 
-feature 'User registers' do
-  scenario 'User registers without invite' do
+feature 'User registers without invite', :js, :vcr do
+  background do
     visit register_path
-    fill_in "Email Address", with: "tom@tom.com"
-    fill_in "Password", with: "password"
-    fill_in "Full Name", with: "Tom Taylor"
+  end
+
+  scenario 'with valid user and card info' do
+    fill_in_user_info(email: "tom@tom.com")
+    fill_in_credit_card_info(card_number: '4242424242424242')
     click_button "Sign Up"
+    sleep 1
```

Comment:
You should avoid `sleep` in your feature specs. The expectation on line 13 should take care of this for you. Capybara will wait up to 2 seconds for an element to be shown by default. If you need longer you can alter that, but 2 seconds should be enough. The key is to expect content that is not on the previous page but is on the next page.